andela/eagle-bn-backend
# #168781795 setup hound CI

#### What does this PR do?
- sets up Hound CI  in this application
#### Description of Task to be completed?

- Add .hound.yml file
- add hound badge in the readme

#### How should this be manually tested?
- clone the repository
- check the presence of .hound.yml file
- check hound badge in the readme

#### Any background context you want to provide?
nan
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/168781795
#### Screenshots (if appropriate)
nan
#### Questions:
nan

[Finishes #168781795]
